python/oneseismic/scan/__main__.py

```diff
@@ -57,8 +58,13 @@ def main(argv):
             secondary = args.secondary_word,
             endian    = endian,
         )
+        geo = GeoScanner(
+            il_word   = args.primary_word,
+            xl_word = args.secondary_word,
```

Comment:
We don't actually require any particular sorting, but we are currently setting the axes labels so that primary word becomes inline number and secondary word becomes crossline number. These variable names should be changed to reflect that.

Reply:
Yes, this is a result of a refactoring that changed the behavior and did not change the variable names accordingly. This should be further investigated to ensure that it is in fact like that and that I'm not missing something, but if it is the case it should be fixed. So I very much agree that something should be done about it, but I think it should be its own PR.